cinema/migrations/0001_initial.py

```diff
@@ -33,18 +48,40 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name="Genre",
             fields=[
-                ("id", models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
                 ("name", models.CharField(max_length=255, unique=True)),
             ],
         ),
         migrations.CreateModel(
             name="Movie",
             fields=[
-                ("id", models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
                 ("title", models.CharField(max_length=255)),
                 ("description", models.TextField()),
                 ("duration", models.IntegerField()),
-                ("image", models.ImageField(null=True, upload_to=cinema.models.movie_image_file_path)),
+                (
+                    "image",
+                    models.ImageField(
+                        null=True,
+                        upload_to=cinema.models.movie_image_file_path,
```

Comment:
Ensure that the `movie_image_file_path` function or variable is correctly defined in the `cinema.models` module. This function should return the path where the image files will be uploaded. If it's not defined, this will cause an error when trying to upload images.